<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CRechnungInfo</class>
 <widget class="QDialog" name="CRechnungInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>1000</height>
   </size>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBoxRechInfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>801</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Kunden und Daten</string>
   </property>
   <widget class="QLabel" name="_Titel">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>50</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Titel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RechDatum">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Kunde Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Organisation">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>130</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Strasse| Nr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_zustaendig">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>180</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>PLZ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MitarbSP">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Ort</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_RechnungNr">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>50</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rechnungsnr</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_MahnungStufe">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Land</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Faelig">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>110</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>FÃ¤lligkeitsdatum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Status">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>140</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Leistungszeitraum">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>80</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rechnungsdatum</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Schlussrechnung">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>170</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Zahlungbedingsgungen</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Mahnhistorie">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>220</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Hinweis: Nachrichte an deinen Kunden</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Title">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_KundeName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Strasse_Nr">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>110</y>
      <width>221</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PLZ">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Ort">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>210</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Land">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>221</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Status">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>140</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Zahlungbedingungsgen">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>170</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Faelligkeitsdatum">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>110</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rechnungsdatum">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>80</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_RechNr">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>50</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mahnhistorie">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>200</y>
      <width>201</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxDetails">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>330</y>
     <width>801</width>
     <height>301</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Rechnungszeilen</string>
   </property>
   <widget class="QLabel" name="label_MengeLager">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>30</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Produkt Name</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_ProduktName">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>50</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Rabatt2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>180</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summenetto">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>30</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Einheit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Menge">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>29</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Preise(netto)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Listenpreis">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Menge</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_PreiseNetto">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Menge">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Einheit">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summenetto_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>30</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mwst</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mwst">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Betrag">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxSumme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>781</width>
      <height>41</height>
     </rect>
    </property>
    <property name="title">
     <string>Zwischensummen ohne Mwst.</string>
    </property>
    <widget class="QLineEdit" name="lineEdit_ZwischBetrag">
     <property name="geometry">
      <rect>
       <x>610</x>
       <y>10</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_Summ">
     <property name="geometry">
      <rect>
       <x>540</x>
       <y>10</y>
       <width>71</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Betrag(netto)</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_Plus">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>30</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(+)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Minus_2">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>180</y>
      <width>16</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(-)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Plus_2">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>210</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(+)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Menge_3">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>180</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Rabatt</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summenetto_3">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>210</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Versandkosten</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Plus_3">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>240</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>(+)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summenetto_4">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Versandkosten Mwst.</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summenetto_5">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Mwst.</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Versandkosten">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>210</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_VersandkostenMwst">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>240</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Mwst2">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>150</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Waehrung">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>30</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>WÃ¤hrung</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBoxWaehrung">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>50</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Gesamt">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>270</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Gesamt</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_GesamtBetrag">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>270</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Gesamt">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>190</y>
      <width>241</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Gesamt Betrag</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSumme_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>1080</y>
     <width>981</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Summe</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_SummeEnde">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>10</y>
      <width>161</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Summ_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>10</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Summe</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSpeAbrClose">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>1120</y>
     <width>981</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Speicher Abbrechen und Close</string>
   </property>
   <widget class="QPushButton" name="pushButtonSpeichern">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbbrecher">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Abbrechern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxSpAbBClose">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>801</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Apeicher Abbrechen Close</string>
   </property>
   <widget class="QPushButton" name="pushButtonSpeichern_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Speichern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAbbrecher_2">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Abbrechern</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose_2">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Close</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonClose_AdminInfo">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>10</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Adminidtration </string>
    </property>
   </widget>
   <zorder>pushButtonSpeichern_2</zorder>
   <zorder>pushButtonAbbrecher_2</zorder>
   <zorder>pushButtonClose_2</zorder>
   <zorder>pushButtonClose_AdminInfo</zorder>
   <zorder>groupBoxRechInfo</zorder>
  </widget>
  <widget class="QLabel" name="label_Listenpreis_3">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>360</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Betrag(netto)</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxKonditionen">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Konditionen</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_Geschaefsbedingungen">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Geschaeftbedingung">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>GeschÃ¤ftsbedingungen</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxZusatzinfo">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>730</y>
     <width>801</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Zusatzinformationen</string>
   </property>
   <widget class="QLineEdit" name="lineEdit_ProdBeschreibung">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>621</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_Beschreibung_2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Beschreibung</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxAdresse">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>630</y>
     <width>801</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="title">
    <string>Kopie</string>
   </property>
   <widget class="QRadioButton" name="radioButtonKopieLiederadresse">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>10</y>
      <width>151</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kopiere Lieferadresse</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButtonKopieRechAdresse">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>181</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Kopiere Rechnungsadresse</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
